analyzer/analyzer.c

```diff
@@ -181,6 +182,7 @@ int main(int argc, char **argv)
     }
     else if (strcasecmp(av[1], "-camel") == 0)
     {
+      showCamel = true;
       camelCase(false);
```

Comment:
Since the term 'camel case' is already used (and it being the proper term), it should be used everywhere. Introducing a second term 'camel' for the same aspect/thing/... is bad. Also 'show camel' sounds weird.

Reply:
This is actually derived from the argument name, which is `-camel`